### PR TITLE
[DOCFIX] Improve troubleshooting doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
A few small improvements on troubleshooting

### CODE REVIEW

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -221,11 +226,10 @@ Tips:
 - You can use Mesos and Yarn integration if you are already using Mesos or Yarn to manage your cluster.
 - If the under storage is remote (like S3 or remote HDFS), using Alluxio can be especially beneficial.
 
-### Q: Why do I see "Unsupported major.minor version 52.0" error when I start Alluxio?
```
Comment (jiacheliu3) [inline docs/en/operation/Troubleshooting.md:224]: This error msg is no longer there

File: docs/en/operation/Basic-Logging.md [hunk 2]
```diff
@@ -37,6 +37,20 @@ See the
 [configuration settings page]({{ '/en/operation/Configuration.html' | relativize_url }}#environment-variables)
 for more information.
 
+By default, the `*.log` files rotate. For example this is the default log4j configuration for master.log:
```
Comment (jiacheliu3) [inline docs/en/operation/Basic-Logging.md:40]: Adding this piece as well, the other page is referring to this part.

File: docs/en/operation/Troubleshooting.md [hunk 3]
```diff
@@ -29,10 +29,15 @@ Workers are doing, especially when running into any issues. If you do not unders
...
 in the case the problem has been discussed before. 
 You can also join our [Slack channel](https://slackin.alluxio.io/) and seek help there.
+You can find more details about the Alluxio server logs [here]({{ '/en/operation/Basic-Logging.html#server-logs' | relativize_url }}).
 
 The client-side logs are also helpful when Alluxio service is running but the client cannot connect to the servers.
 Alluxio client emits logging messages through log4j, so the location of the logs is determined by the client side
 log4j configuration used by the application.
+You can find more details about the client-side logs [here]({{ '/en/operation/Basic-Logging.html#application-logs' | relativize_url }}).
+
+The user logs in `${ALLUXIO_HOME}/logs/user/` are the logs from running Alluxio shell.
```
Comment (LuQQiu) [inline docs/en/operation/Troubleshooting.md:39]: the user logs can also be users using other native APIs or FileSystem APIs? not just Alluxio shell?

**Review by LuQQiu — APPROVED** [review]
Thanks for improving the doc! Left one minor comments, LGTM

File: docs/en/operation/Troubleshooting.md (hunk 3 above)
Comment (jiacheliu3) [reply docs/en/operation/Troubleshooting.md:39]: I believe with FileSystem API, the user will define their own logger logging to their own specified location. The FileSystem code API based code should count as "client application".

In our code `logs/user/user_*.log` is only used in ALLUXIO_USER_JAVA_OPTS, which seems to be specified for shell as https://github.com/Alluxio/alluxio/blob/6b35ea2d826dd79b5c9c60ab4d5317e8bdc49a74/conf/alluxio-env.sh.template#L53

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Comment by jiacheliu3** [comment]
@LuQQiu need your permission?

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**